nix/proto-lens-pkg.nix

```diff
@@ -0,0 +1,23 @@
+{ mkDerivation, base, bytestring, containers, deepseq, ghc-prim
+, hpack, lens-family, parsec, pretty, primitive, profunctors
```

Comment:
I guess  haskell-language-server-wrapper is available in any distro. For example in arch it downloads hls for every version of ghc. Autocomplete and stuff is working in protolens repo without any additional configuration and problems

Reply:
I'll remove hie.yaml from this branch before opening PR to google. Let's keep it in our master, it will not hurt